src/taco-dependency-installer/installerRunner.ts

```diff
@@ -122,8 +122,13 @@ class InstallerRunner {
 
                 // Verify that the destination folder is empty if it already exists
                 if (fs.existsSync(value.installDestination)) {
-                    if (fs.readdirSync(value.installDestination).length > 0) {
-                        throw new Error(resources.getString("PathNotEmpty", value.displayName, value.installDestination));
+                    var dirItems: string[] = fs.readdirSync(value.installDestination);
+
+                    if (dirItems.length > 0) {
```

Comment:
I was planning on implementing the "Patch Up" feature/suggestion bug in a separate changeset. This is just to make sure we don't crash / throw error when folder only contains a .DS_Store